<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChromaDecoderConfigDialog</class>
 <widget class="QDialog" name="ChromaDecoderConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chroma Decoder Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="outerVerticalLayout">
   <item>
    <widget class="QLabel" name="chromaGainLabel">
     <property name="text">
      <string>Chroma gain:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="chromaGainHorizontalSlider">
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="chromaGainValueLabel">
     <property name="text">
      <string>1.0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="chromaPhaseLabel">
     <property name="text">
      <string>Chroma phase:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="chromaPhaseHorizontalSlider">
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="chromaPhaseValueLabel">
     <property name="text">
      <string>0.0Â°</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="yNRLabel">
     <property name="text">
      <string>Luma noise reduction:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSlider" name="yNRHorizontalSlider">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="yNRValueLabel">
     <property name="text">
      <string>0.00 IRE</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="standardTabs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="palTab">
      <attribute name="title">
       <string>PAL</string>
      </attribute>
      <layout class="QVBoxLayout" name="palVerticalLayout">
       <item>
        <widget class="QLabel" name="palFilterLabel">
         <property name="text">
          <string>Chroma filter:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="palFilterPalColourRadioButton">
         <property name="text">
          <string>PalColour 2D</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">palFilterButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="palFilterTransform2DRadioButton">
         <property name="text">
          <string>Transform PAL 2D</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">palFilterButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="palFilterTransform3DRadioButton">
         <property name="text">
          <string>Transform PAL 3D</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">palFilterButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="thresholdModeCheckBox">
         <property name="text">
          <string>Use threshold comparison mode</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="thresholdLabel">
         <property name="text">
          <string>Transform threshold:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="thresholdHorizontalSlider">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="thresholdValueLabel">
         <property name="text">
          <string>0.4</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="showFFTsCheckBox">
         <property name="text">
          <string>Overlay FFT visualisation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="simplePALCheckBox">
         <property name="text">
          <string>Use Simple PAL decoder</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ntscTab">
      <attribute name="title">
       <string>NTSC</string>
      </attribute>
      <layout class="QVBoxLayout" name="ntscVerticalLayout">
       <item>
        <widget class="QLabel" name="ntscFilterLabel">
         <property name="text">
          <string>Chroma filter:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="ntscFilter1DRadioButton">
         <property name="text">
          <string>1D</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">ntscFilterButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="ntscFilter2DRadioButton">
         <property name="text">
          <string>2D</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">ntscFilterButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="ntscFilter3DRadioButton">
         <property name="text">
          <string>3D</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">ntscFilterButtonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_9">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QCheckBox" name="adaptiveCheckBox">
         <property name="text">
          <string>Enable adaptive filter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showMapCheckBox">
         <property name="text">
          <string>Overlay adaptive filter visualisation</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="colorLpfCheckBox">
         <property name="text">
          <string>Use chroma post-filter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="colorLpfHqCheckBox">
         <property name="text">
          <string>Use full bandwidth Q channel filter</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="cNRLabel">
         <property name="text">
          <string>Chroma noise reduction:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="cNRHorizontalSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="cNRValueLabel">
         <property name="text">
          <string>0.00 IRE</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="ntscFilterButtonGroup"/>
  <buttongroup name="palFilterButtonGroup"/>
 </buttongroups>
</ui>
